src/Centreon/Domain/Repository/TimePeriodRepository.php

```diff
@@ -161,4 +161,34 @@ public function getChainByParant($id, &$result)
 
         return $result;
     }
+
+    /**
+     * Get chain of included templates
+     *
+     * @param int $id
```

Comment:
For comments, it would be nice to put more information. Because here, we have just the same thing as what's in the line of code, the parameter is everything. We have no explanation of what it is exactly and what it corresponds to.

Reply:
done